dash/include/dash/memory/GlobStaticMem.h

```diff
@@ -277,13 +277,14 @@ class GlobStaticMem
   /**
    * Copy-assignment operator.
    *
-   * \TODO make move constructor defaultable by using RAII 
    */
   self_t & operator=(const self_t & rhs)
     = delete;
 
   /**
    * Move-assignment operator.
+   *
+   * \TODO make move constructor defaultable by using RAII 
    */
   self_t & operator=(self_t && other) {
```

Comment:
Arg... these annoying missing return statements. Will fix it. With defaulting I mean writing  `self_t & operator=(self_t && other) = default`. This is only possible if all members support moving them. Pointers which will be deallocated on destruction have to be encapsulated with `std::unique_ptr`, as it is set to `nullptr` after moving it away.